src/components/GalleryBrick/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import FontAwesome from 'react-fontawesome';
+import styles from './styles';
+
+class GalleryBrick extends Component {
+  constructor(props){
+    super(props);
+    this.state = { currentPage: 1 }
+  }
+
+  calculatePages(){
+    let { items , itemsPerPage } = this.props;
+    let pages = parseInt(items.length / itemsPerPage);
+
+    return (items.length % itemsPerPage > 0) ? pages + 1 : pages;
+  }
+
+  renderItems(item, key, s){
+    return (
+      <div style={ s.imgGallery }>
+        <img key={ key } style={ s.image} src={ item } />
+      </div>
+      )
+  }
+
+  renderPagesItems(items, s){
```

Comment:
- i like better the name renderPageOfItems
- use the complete name for param 's', call it 'styles'